### PR TITLE
[automation/nodejs,go,python] - Expose pulumiVersion as string

### PR DESCRIPTION
**Pull request by komalali**
Follow-up from feedback on https://github.com/pulumi/pulumi/pull/6590

### CODE REVIEW

**Review by stack72 — APPROVED** [review]

File: sdk/go/x/auto/workspace.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ type Workspace interface {
 	// This customizes the location of $PULUMI_HOME where metadata is stored and plugins are installed.
 	PulumiHome() string
 	// PulumiVersion returns the version of the underlying Pulumi CLI/Engine.
-	PulumiVersion() semver.Version
+	PulumiVersion() string
```
Comment (justinvp) [inline sdk/go/x/auto/workspace.go:79]: Can the import at to the top of the file be removed?

File: sdk/nodejs/x/automation/workspace.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export interface Workspace {
     /**
      * The version of the underlying Pulumi CLI/Engine.
      */
-    readonly pulumiVersion: semver.SemVer;
+    readonly pulumiVersion: string;
```
Comment (justinvp) [inline sdk/nodejs/x/automation/workspace.ts:46]: Similarly can the import in this file be removed?

File: sdk/python/lib/pulumi/x/automation/_workspace.py [hunk 3]
```diff
@@ -132,7 +132,7 @@ class Workspace(ABC):
     Environment values scoped to the current workspace. These will be supplied to every Pulumi command.
     """
 
-    pulumi_version: VersionInfo
+    pulumi_version: str
```
Comment (justinvp) [inline sdk/python/lib/pulumi/x/automation/_workspace.py:135]: Can the import in this file be removed?

**Review by justinvp — APPROVED** [review]
LGTM

**Pull request merged**